concent-vm/install-concent.yml

```diff
@@ -0,0 +1,90 @@
+- hosts:
+    - vagrant
+  vars_files:
+    - ../containers/versions.yml
+    - ../concent-builder/repositories.yml
+    - vars.yml
+  tasks:
+    - become:      yes
+      become_user: vagrant
+      block:
+        - name:   Clone concent repository
+          git:
+            repo:    "{{ repositories.concent.url }}"
+            dest:    "{{ concent_dir }}"
+            version: "{{ concent_version }}"
```

Comment:
The issue states that this should check out code from `master` by default. As it is now, it'll use version from `versions.yml`.

It's fine but you should have mentioned this change in a comment in the issue.